Towny/src/main/java/com/palmergames/bukkit/towny/object/TownyWorld.java

```diff
@@ -1035,6 +1038,15 @@ public boolean isFriendlyFireEnabled( ) {
 		return isFriendlyFire;
 	}
 
+	public void setJailing(boolean parseBoolean) {
+		isJailing = parseBoolean;
+		
+	}

```

Comment:
setJailingEnabled would read better to me and would also match with the other method, the field would also need to be renamed.

There's also an extra line of whitespace that should be removed